addons/mail/models/mail_thread.py

```diff
@@ -1469,6 +1469,8 @@ def _message_add_suggested_recipient(self, result, partner=None, email=None, lan
             return result
         if partner and partner.id in [val[0] for val in result[self.ids[0]]]:  # already existing partner ID -> skip
             return result
+        if lang is not None and not self.env['res.lang']._lang_get(lang):
```

Comment:
```suggestion
        if lang and not self.env['res.lang']._lang_get(lang):
```
so `False` gets treated the same (in https://github.com/odoo/odoo/commit/2625132dd795bbeb52e61d5704e3b19d61f90a83 `lead.lang_code` will give False as attribute value if lang_code is not set)